### PR TITLE
Jcaccamo/surface placement rel to scene

### PR DESCRIPTION
**Pull request by jared-2016**
@ldanzinger @JamesMBallard Ready for review. Thank you!

Updating surface placement sample to include the new design which includes relative to scene.

### CODE REVIEW

File: ArcGISRuntimeSDKQt_CppSamples/Scenes/SurfacePlacement/README.md [hunk 1]
```diff
@@ -6,37 +6,38 @@ Position graphics relative to a surface using different surface placement modes.
...
+    * `RelativeToScene`, position graphic using its Z value plus the altitude values of the scene.
 2. Add graphics to the graphics overlay, `GraphicsOverlay::graphics()::append(Graphic)`.
 3. Add each graphics overlay to the scene view by calling `SceneView::graphicsOverlays()::append(overlay)`.
 
 ## Relevant API
 
 * Graphic
 * GraphicsOverlay
-* LayerSceneProperties::SurfacePlacement
+* LayerScenePropertie::surfacePlacement
```
Comment (JamesMBallard) [inline ArcGISRuntimeSDKQt_CppSamples/Scenes/SurfacePlacement/README.md:30]:
```suggestion
* LayerSceneProperties::surfacePlacement
```

File: ArcGISRuntimeSDKQt_CppSamples/Scenes/SurfacePlacement/README.md [hunk 2]
```diff
@@ -6,37 +6,38 @@ Position graphics relative to a surface using different surface placement modes.
...
 
 ## How to use the sample
 
-The application loads a scene showing three points that use the individual surface placement rules (Absolute, Relative, and either Draped Billboarded or Draped Flat). Use the control to toggle the draped mode, then explore the scene by zooming in/out and by panning around to observe the effects of the surface placement rules.
+The application loads a scene showing four points that use individual surface placement modes (Absolute, Relative, Relative to Scene, and either Draped Billboarded or Draped Flat). Use the toggle to change the draped mode and the slider to dynamically adjust the Z value of the graphics. Explore the scene by zooming in/out and by panning around to observe the effects of the surface placement rules.
 
 ## How it works
 
-1. Create a `GraphicsOverlay` for each placement mode, setting `SceneProperties::SurfacePlacement`:
+1. Create a `GraphicsOverlay` for each placement mode, setting `SceneProperties::surfacePlacement`:
```
Comment (JamesMBallard) [inline ArcGISRuntimeSDKQt_CppSamples/Scenes/SurfacePlacement/README.md:17]:
```suggestion
1. Create a `GraphicsOverlay` for each placement mode, setting `LayerSceneProperties::surfacePlacement`:
```

File: ArcGISRuntimeSDKQt_CppSamples/Scenes/SurfacePlacement/Surface_Placement.qml [hunk 3]
```diff
@@ -59,5 +65,79 @@ SurfacePlacementSample {
...
+                        width: 20
+                        height: width
+                    }
+                    Rectangle {
+                        id: headingHandleRect
+                        height: childrenRect.height
+                        width: childrenRect.width
+                        radius: 3
+                        x: headingHandleNub.x - width
+                        y: headingHandleNub.y - height/2 + headingHandleNub.height/2
```
Comment (JamesMBallard) [inline ArcGISRuntimeSDKQt_CppSamples/Scenes/SurfacePlacement/Surface_Placement.qml:125]: Consistent spacing
```suggestion
                        y: headingHandleNub.y - height / 2 + headingHandleNub.height / 2
```

File: ArcGISRuntimeSDKQt_QMLSamples/Scenes/SurfacePlacement/Surface_Placement.qml [hunk 4]
```diff
@@ -214,12 +247,108 @@ Rectangle {
...
+                        width: 20
+                        height: width
+                    }
+                    Rectangle {
+                        id: headingHandleRect
+                        height: childrenRect.height
+                        width: childrenRect.width
+                        radius: 3
+                        x: headingHandleNub.x - width
+                        y: headingHandleNub.y - height/2 + headingHandleNub.height/2
```
Comment (JamesMBallard) [inline ArcGISRuntimeSDKQt_QMLSamples/Scenes/SurfacePlacement/Surface_Placement.qml:306]:
```suggestion
                        y: headingHandleNub.y - height / 2 + headingHandleNub.height / 2
```

**Comment by jared-2016** [comment]
@JamesMBallard review changes applied. Thanks for catching those!

**Pull request merged**